server/db/add_client.sh

```diff
@@ -10,9 +10,9 @@ if [[ $EUID > 0 ]]; then
     exit 1
 elif [[ ! -n "$6" ]]; then
     echo ""
-    echo "Usage: $0 path_to_.env_file new_client_name responder_phone_numbers fallback_phone_numbers from_phone_number path_to_buttons_csv"
+    echo "Usage: $0 path_to_.env_file new_client_name responder_phone_numbers fallback_phone_numbers from_phone_number path_to_buttons_csv locationid"
```

Comment:
Locationid shouldn't be here; leave this line unchanged

Reply:
fixed this!